Makefile

```diff
@@ -1,5 +1,6 @@
 VERSION = $(shell cat emscripten-version.txt | sed s/\"//g)
-DESTDIR ?= ../emscripten-$(VERSION)
+TMPDESTDIR := $(shell mktemp -d)
+DESTDIR ?= $(TMPDESTDIR)/emscripten-$(VERSION)
```

Comment:
I still don't understand that problem this is solving.  Can you explain?

Reply:
`../emscripten-$(VERSION)` is exactly the same as `$PWD/emscripten-$(VERSION)` when you first extracted from zip
It will end up delete itself down the line...

Reply:
I see, but as long as you run with `make DESTDIR=xxx` it should be fine right?

Should we just error out if $DESTDIR already exists instead of deleting it?

Reply:
I feel that running just `make` and errors out is wrong, lol
and `tools/install.py` has that error implemented so it must point to folder that is clean

Reply:
Actually now I am not using this Makefile directly anymore as I opted to use the patch `install.py`. I just want to raise this issue whether it make sense to still use this Makefile from the "Releases" zips...

Reply:
Ok I suppose that's the way I guess
The archlinux guys seem to package using `make install DESTDIR=xxx` and from the git repo
https://github.com/archlinux/svntogit-community/blob/packages/emscripten/trunk/PKGBUILD

I think I will try to change from zip archive to git repo source somehow...